README.rst

```diff
@@ -92,6 +94,24 @@ etc., we provide `Docker <https://docker.com/>`_ images where we've
 done the work for you. The images are uploaded to `quay.io`_ and are tagged
 for repeatable builds.
 
+manylinux_2_34 (AlmaLinux 9 based)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Toolchain: GCC 13
```

Comment:
Didn't you say in #1585 that you'd want to start off with GCC 14?

Reply:
I said that "I'd like to wait" but if I don't get some sort of planning/roadmap for availability (it's been in stream 9 for a while now but the fact that it's nowhere else requires more investigation on why), we might want to start with 13 & update later on.

Reply:
Digging a bit more into this I only could find that there's some on-going work in RHEL bug tracker. It seems they're already using it in a staging area but still finding bugs and reworking if I understand correctly. I have not found any kind of planning around this.

Reply:
Makes sense. 👌

We're also using GCC 13 in conda-forge until GCC 15 is released, for pretty much those reasons (picking up all the bugfixes over the course of a year leading up to the next major version).

Reply:
I can't speak to the Alma roadmap, but in general they try to match RHEL.  RHEL doesn't make exact dates public, but there is a public [planning guide](https://access.redhat.com/support/policy/updates/errata#RHEL9_Planning_Guide) that should give you a rough idea.  New packages such as GCC 14 are typically delivered in CentOS Stream once they pass QA, and then show up in the next RHEL minor version.  Those minor versions happen about every six months (since RHEL 8, before that it was not consistent).  So the next time you see a feature in CentOS Stream and want to know when it will be in RHEL, look back at the [release date](https://access.redhat.com/articles/3078) for the last minor version, and estimate six months after that.

Reply:
9.5 has been released and includes gcc-toolset-14